rust/tests/tests/test_complex_struct.rs

```diff
@@ -60,7 +60,7 @@ fn complex_struct() {
         op: Some("option".to_string()),
         op2: None,
         date: NaiveDate::from_ymd_opt(2025, 12, 12).unwrap(),
-        time: NaiveDateTime::from_timestamp_opt(1689912359, 0).unwrap(),
+        time: DateTime::from_timestamp(1689912359, 0).unwrap().naive_utc(),
```

Comment:
The semantic doesn't changes: https://docs.rs/chrono/latest/chrono/struct.NaiveDateTime.html#method.from_timestamp_opt

The previous call `NaiveDateTime::from_timestamp_opt` also returns UTC datetime

>Makes a new NaiveDateTime corresponding to a UTC date and time, from the number of non-leap seconds since the midnight UTC on January 1, 1970 (aka “UNIX timestamp”) and the number of nanoseconds since the last whole non-leap second.